blog/_posts/2021-07-06-distributed-rdataframe.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "RDataFrame is going distributed!"
+layout: archive
+author: Vincenzo Eduardo Padulano
+---
+
```

Comment:
please make the first paragraph a short, self-contained intro: it's what appears in the blog preview at https://root.cern